.github/workflows/build-wheels.yml

```diff
@@ -0,0 +1,101 @@
+name: Build wheels
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '0 0 * * 0,3' # 2/weekly
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+        os: [ubuntu-latest, macos-latest]
+        platform: [x64]
+        PLAT: [x86_64]
+    env:
+      PKG_NAME: gensim
+      BUILD_COMMIT: 4.0.0beta
+      UNICODE_WIDTH: 32
+      MB_PYTHON_VERSION: ${{ matrix.python-version }}
+      TRAVIS_PYTHON_VERSION: ${{ matrix.python-version }}
```

Comment:
Why do we need separate TRAVIS_ environment variables here? We aren't using Travis anymore.

Some of these values are redundant (e.g. MB_PYTHON_VERSION and TRAVIS_PYTHON_VERSION). Some of them are already available via the default environment variables exposed by Github Actions. Let's only keep the ones that we _really_ need.

Reply:
(if multibuild expects those variables to be there, it's definitely worth documenting that)